app/models/vita_partner.rb

```diff
@@ -27,52 +28,19 @@
 #  fk_rails_...  (coalition_id => coalitions.id)
 #
 class VitaPartner < ApplicationRecord
-  belongs_to :coalition, optional: true
   has_many :clients
   has_many :intakes
   has_many :source_parameters
-  has_many :serviced_zip_codes, class_name: "VitaPartnerZipCode"
-  has_many :serviced_states, class_name: "VitaPartnerState"
-  belongs_to :parent_organization, class_name: "VitaPartner", optional: true
-  has_one :organization_capacity
-  validate :one_level_of_depth
-  validate :no_coalitions_for_sites
-  validate :no_capacity_for_sites
-  validate :no_allows_greeters_for_sites
-  validates :name, uniqueness: { scope: [:coalition, :parent_organization] }
-  validates :capacity_limit, gyr_numericality: { greater_than_or_equal_to: 0 }, allow_blank: true
 
-  scope :organizations, -> { where(parent_organization: nil) }
-  scope :sites, -> { where.not(parent_organization: nil) }
-  has_many :child_sites, -> { order(:id) }, class_name: "VitaPartner", foreign_key: "parent_organization_id"
-  scope :allows_greeters, -> {
-    greetable_organizations = organizations.where(allows_greeters: true)
-    greetable_sites = sites.where(parent_organization: greetable_organizations)
-    greetable_organizations.or(greetable_sites)
+  scope :organizations, -> { Organization.default_scoped.with_child_sites }
```

Comment:
If you change this to `VitaPartner.where(type: Organization::TYPE)` you may have an easier time. Why do `with_child_sites`?

Reply:
If `.with_child_sites` was always called on `VitaPartner` then it's safe to call for both `Organization` and `Site`. I suggest that if you're working around a difficulty where the two queries you're `or`ing aren't the same "shape".